addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3486,7 +3486,8 @@ var SnippetsMenu = Widget.extend({
             $(selection.focusNode).parent('[data-oe-model]:not([data-oe-type="html"]):not([data-oe-field="arch"])').length ||
             (e && $(e.target).closest('.fa, img').length ||
             this.options.wysiwyg.lastMediaClicked && $(this.options.wysiwyg.lastMediaClicked).is('.fa, img')) ||
-            (this.options.wysiwyg.lastElement && !this.options.wysiwyg.lastElement.isContentEditable)
+            (this.options.wysiwyg.lastElement && !this.options.wysiwyg.lastElement.isContentEditable) ||
```

Comment:
Note: funny, actually in 15.4, the behavior for Firefox was not losing the text tools but keeping the selection on the text when using column options, which seems better to me. But indeed Chrome lost the selection and the text tools. I guess it's not bad unifying the behavior, I guess it's not possible for Chrome to keep the selection instead :grimacing:

Reply:
With https://github.com/odoo/odoo/commit/31cc10b91dc7762e23b4bde9b945be0c4ce3fe3b, because there is now one document for the SnippetsMenu, and another one for the snippets, it is possible to keep the text selection on chrome as well. 
We actually added extra code to keep the Chrome behaviour (which I thought was standard) => lose the focus on text when using snippets options. I removed that code in this commit (which is still WIP) : https://github.com/odoo/odoo/pull/96401/commits/a26be790a2a5b2bfc717abbfc4c983ff2cb600bb
Tests are green, maybe we can drop this commit and keep the linked one ? If so, should the text be blurred on clicks on discard/save buttons ?

Reply:
Indeed, I prefer the old Firefox behavior if possible :+1: It make sense to me that if you select text, you want to keep the selection until you decide to remove it yourself. So yes, drop this commit :+1: